webapp/tests/karma/ts/modules/reports/reports.component.spec.ts

```diff
@@ -66,7 +78,10 @@ describe('Reports Component', () => {
       isOnlineOnly: sinon.stub()
     };
     datePipe = { transform: sinon.stub() };
-
+    userContactService = {
+      get: sinon.stub().resolves(userContactDoc),
+      getCurrentLineageLevel : sinon.stub().resolves('parent')
```

Comment:
Should include tests to cover the various new logic.  Make sure that the current lineage level for the user is filtered out from the displayed lineage (but only when the user is offline).